lib-extra/src/main/java/com/diffplug/spotless/extra/java/EclipseFormatterStep.java

```diff
@@ -47,14 +40,14 @@ private EclipseFormatterStep() {}
 	private static final String FORMATTER_METHOD = "format";
 
 	/** Creates a formatter step for the given version and settings file. */
-	public static FormatterStep create(File settingsFile, Provisioner provisioner) {
-		return create(defaultVersion(), settingsFile, provisioner);
+	public static FormatterStep create(Iterable<File> settingsFiles, Provisioner provisioner) {
```

Comment:
The common interface, is actually imposed on the user interfaces. How `plugin-maven` will implement this interface is not known yet, but for plugin-gradle we once said that we stick with the existing methods and basically change `File` to `File...`. So newer versions of `plugin-gradle` 
will no use `FormatterStep create(File settingsFile, ...`. So currently this method has no user (I don't count tests), and I see currently no use case. Hence it is dead code and should be removed. If you want to keep it for binary compat, it should fade out, so it should be marked deprecated.